.github/workflows/build_test.yml

```
@@ -78,11 +77,44 @@ jobs:
           ref: ${{ fromJSON(steps.pr.outputs.result) }}
           fetch-depth: 0
 
+      - name: Validate PR title and commit message
+        shell: bash
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)